lib/resources/r4/communication.yaml

```diff
@@ -23,7 +22,7 @@ fields:
   description: The status of the Communication.
   url: https://hl7.org/fhir/r4/communication-definitions.html#Communication.status
   action: create
-  note: As Millenniums Message Center Communicaitons are instantaneously delivered, all communications must be in a "completed" status
+  note: All Communications must be in 'completed' status.
   example: |
     {
       "status": "completed"
```

Comment:
Can you beautify the category example? Just steal it from the JSON I posted on Yuyu's

Reply:
https://github.com/cerner/fhir.cerner.com/pull/435/commits/dd36f4e03a4868b2986ff9e2dc706e8a11e22519#diff-f08ae1e4c063b9a36a22adaa43fecf21L40-R51